Correct GLib version check on g_set_str
Patch by Simon Branford (University of Birmingham)
--- gdk/gdkprivate.h.orig	2023-07-24 20:41:16.096936629 +0100
+++ gdk/gdkprivate.h	2023-07-24 20:41:24.875062247 +0100
@@ -22,7 +22,7 @@
 #define I_(string) g_intern_static_string (string)
 #endif
 
-#if !GLIB_CHECK_VERSION (2, 75, 1)
+#if !GLIB_CHECK_VERSION (2, 75, 0)
 static inline gboolean
 g_set_str (char       **str_pointer,
            const char  *new_str)
